authorize_request_handler_oidc_request_test.go

```diff
@@ -190,18 +182,6 @@ func TestAuthorizeRequestParametersFromOpenIDConnectRequest(t *testing.T) {
 			client:     &DefaultOpenIDConnectClient{JSONWebKeysURI: reqJWK.URL, RequestObjectSigningAlgorithm: "RS256", RequestURIs: []string{reqTS.URL}},
 			expectForm: url.Values{"response_type": {"token"}, "response_mode": {"post_form"}, "scope": {"foo openid"}, "request_uri": {reqTS.URL}, "foo": {"bar"}, "baz": {"baz"}},
 		},
-		{
-			d:          "should pass when request object uses algorithm none",
```

Comment:
A particular reason for removing these? These are neccessary for some OIDC flows :/

Reply:
I removed them because go-jose [does not support](https://github.com/square/go-jose/issues/185) alg:none therefore tests trying to check this functionality where removed. I assumed it was just a sanity check, alg:none is seen as insecure and a door for vulnerabilities. But if required I can provide that functionality. It is just to base64-urlencoded `header.claims.empty-signature`, right?

Reply:
Yeah, unfortunately it's still supported as an algorithm in OIDC for whatever reason (probably some legacy corp not being able to implement JWT with algs) - let's see the result of https://github.com/ory/hydra/pull/2526/checks?check_run_id=2554759989 and if it passes we can leave this removed